src/metabase/db/custom_migrations.clj

```diff
@@ -1396,3 +1396,26 @@
 (define-reversible-migration MigrateMetricsToV2
   (metrics-v2/migrate-up!)
   (metrics-v2/migrate-down!))
+
+(defn- raw-setting-value [key]
+  (some-> (t2/query-one {:select [:value], :from :setting, :where [:= :key key]})
+          :value
+          encryption/maybe-decrypt))
+
+(define-reversible-migration MigrateUploadsSettings
+  (when-let [db-id (some-> (raw-setting-value "uploads-database-id") Integer/parseInt)]
+    (let [uploads-table-prefix (raw-setting-value "uploads-table-prefix")
+          uploads-schema-name  (raw-setting-value "uploads-schema-name")]
+      (t2/query {:update :metabase_database
+                 :set    {:uploads_enabled      true
+                          :uploads_table_prefix uploads-table-prefix
+                          :uploads_schema_name  uploads-schema-name}
+                 :where  [:= :id db-id]})))
+  (when-let [db (t2/query-one {:select [:*], :from :metabase_database, :where :uploads_enabled})]
```

Comment:
what do you think about cases where we have more than one upload db here?

Reply:
There cannot be more than one DB with `uploads_enabled`. This invariant is maintained by the toucan hooks
https://github.com/metabase/metabase/blob/be10fd4c129a53df86ea49d777181d914db052d6/src/metabase/models/database.clj#L259-L264